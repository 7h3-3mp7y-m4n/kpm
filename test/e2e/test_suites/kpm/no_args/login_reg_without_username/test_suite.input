kpm registry login -p aaaa ghcr.io