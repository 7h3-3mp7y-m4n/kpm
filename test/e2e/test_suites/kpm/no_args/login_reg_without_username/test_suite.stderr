kpm: failed to login, please check registry, username and password is valid.